modules/installing-oadp-rosa-sts.adoc

```diff
@@ -0,0 +1,253 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-oadp-rosa-sts_{context}"]
+= Installing the OADP Operator and providing the IAM role
+
+AWS Security Token Service (AWS STS) is a global web service that provides short-term credentials for IAM or federated users. {product-title} (ROSA) with {sts-short} is the recommended credential mode for ROSA clusters. This document describes how to install {oadp-first} on ROSA with {aws-short} {sts-short}.
+
+
+[IMPORTANT]
+====
+Restic and Kopia are not supported in the OADP on ROSA with {aws-short} {sts-short} environment. Verify that the Restic and Kopia node agent is disabled.
+For backing up volumes, OADP on ROSA with {aws-short} {sts-short} supports only native snapshots and Container Storage Interface (CSI) snapshots.
+====
+
+[IMPORTANT]
+====
+In an Amazon ROSA cluster that uses STS authentication, restoring backed-up data in a different {aws-short} region is not supported.
+
+The Data Mover feature is not currently supported in ROSA clusters. You can use native {aws-short} S3 tools for moving data.
+====
+
+.Prerequisites
+
+* An {product-title} ROSA cluster with the required access and tokens. For instructions, see the previous procedure _Preparing AWS credentials for OADP_. If you plan to use two different clusters for backing up and restoring, you must prepare {aws-short} credentials, including `ROLE_ARN`, for each cluster.
+
+
+.Procedure
+
+. Create an {product-title} secret from your {aws-short} token file by entering the following commands:
+
+.. Create the credentials file:
++
+[source,terminal]
+----
+$ cat <<EOF > ${SCRATCH}/credentials
+[default]
+role_arn = ${ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+----
+
+.. Create a namespace for OADP:
++
+[source,terminal]
+----
+$ oc create namespace openshift-adp
+----
+
+.. Create the {product-title} secret:
++
+[source,terminal]
+----
+$ oc -n openshift-adp create secret generic cloud-credentials \
+  --from-file=${SCRATCH}/credentials
+----
++
+[NOTE]
+====
+In {product-title} versions 4.14 and later, the OADP Operator supports a new standardized {sts-short} workflow through the Operator Lifecycle Manager (OLM)
+and Cloud Credentials Operator (CCO). In this workflow, you do not need to create the above
+secret, you only need to supply the role ARN during link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/user-tasks#olm-installing-from-operatorhub-using-web-console_olm-installing-operators-in-namespace[the installation of OLM-managed operators using the {product-title} web console].
+The preceding secret is created automatically by CCO.
+====
+
+. Install the OADP Operator:
+.. In the {product-title} web console, browse to *Operators* -> *OperatorHub*.
```

Comment:
@CarmiWisemon If we are installing the Operator via UI(Operatorhub) on 4.14+ cluster then it requires role_ARN field. Sorry I just noticed it. 

![sts](https://github.com/openshift/openshift-docs/assets/66882206/07f1ef32-6cd0-4b7c-a394-0bf462e2af5b)